docker/Dockerfile

```diff
@@ -36,14 +36,14 @@ RUN for key in \
     A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
     B9E2F5981AA6E0CD28160D9FF13993A75599653C \
     ; do \
-        # Let's try several servers to ensure we get all the keys
-        for server in \
-            keys.openpgp.org \
-            keyserver.ubuntu.com \
-            hkps.pool.sks-keyservers.net \
-        ; do \
-	        gpg2 --batch --keyserver "$server" --recv-keys "$key"; \
-        done \
```

Comment:
I'd prefer if we leave this nested `for` code block indented, as it makes it easier to read/ troubleshoot.

Reply:
This was auto formatted for me, with what I assume is the defaults.

Reply:
I suspect the autoformatter you're using is not able to detect it's a nested for.

Reply:
Yes that is likely the case (or it's preference is to not nest them), I guess my point is that I wouldn't want to give up formatting for the whole file (if I can't disable this one behavior).